docs/docsite/rst/network/user_guide/platform_nxos.rst

```diff
@@ -122,3 +122,37 @@ Example NX-API Task
 In this example the ``proxy_env`` variable defined in ``group_vars`` gets passed to the ``environment`` option of the module used in the task.
 
 .. include:: shared_snippets/SSH_warning.txt
+
+***********************************
+Cisco Nexus Platform Support Matrix
+***********************************
+
+The following platforms and software versions have been certified to work with this version of Ansible.
```

Comment:
Is this RH certified or Cisco-certified for these platforms and versions?

Reply:
This is mostly a statement about Cisco certification but they are also certified/validated by RH.

Reply:
Maybe something like:
```suggestion
The following platforms and software versions have been certified by Cisco to work with this version of Ansible.
```